### DISCUSSION TITLE
Get basemap

### DISCUSSION THREAD

**Discussion by ai-perture** [EY-Data-Science-Program/2022-Better-Working-World-Data-Challenge#7] [Q&A]
Hello, 
I'm trying to change the region used in challenge one. How can I get the contextily basemap for my new region ? I'm using 
min_lon, min_lat = (152.6, -29.0)  # Lower-left corner
max_lon, max_lat = (153.6, -28)  # Upper-right corner
given in the text file challenge_1_test_regions
Thanks!
Ai-Ling Bonnet

**Comment by BillyDodds** [comment]
Hi @ai-perture, 

Are you using the plotting functions provided in the notebook? They should produce the basemap for you. The specific line of code is `cx.add_basemap(ax, crs={'init':'epsg:4326'})`, where `ax` is the matplotlib axis returned by `fig, ax = plt.subplots()`.

**Reply by ai-perture** [reply]
Hi @BillyDodds, 
It seems it was actually an issue with Azure. It works well today. The error message was " Could not find a suitable TLS CA certificate bundle, invalid path ".
